der/Cargo.toml

```
@@ -16,6 +16,7 @@ edition = "2021"
 rust-version = "1.57"
 
 [dependencies]
+flagset = { version = "^0.4.3", optional = true }
```

As I mentioned on #414, the `^` is the default range matching operator for Cargo:

```suggestion
flagset = { version = "0.4.3", optional = true }
```

---

Done.